### ISSUE TITLE
Sitewide: heading levels

### ISSUE THREAD

**Issue by dlowenberg** [CDL-Dryad/dryad-product-roadmap#594]
Fix heading levels. Add heading tags to the reasons to use Dryad. E.g., <h3>Any field. Any format.</h3>

- Homepage
- Explore Data: Consider having "Explore and find the data you need" as the <h1> for the Search Results page. "Find by" and "Find by location" should be <h2> headings. "Placename" (consider "Geographical location"), "Subject," and "Journal" should be <h3> headings.
- User Dashboard: Make "In Progress" and "Submitted" <h2>s to enable faster navigation. 
- Browse/Search Results: Change the <h3> (if that is indeed the correct heading level to be using here) to only include the linked text. Activating the search result will also expand the selection – or consider having the search results expanded by default.  Also, Consider adding "Search Results" as the <h1> for the search results page. Remove <h3> headings from search facet headings.
- Describe Dataset: Change all of these sections to <h2>s. Consider having the page be fully expanded upon load, rather than having the buttons expand the two optional areas.
